src/entities/SurveyTopic/types.ts

```diff
@@ -0,0 +1,23 @@
+export enum ReactionType {
+  LOVE = 'love',
+  LIKE = 'like',
+  NEUTRAL = 'neutral',
+  CONCERNED = 'concerned',
+  EMPTY = 'empty',
+}
+
+export type SurveyTopicAttributes = {
+  id: string
+  topic_id: string
+  created_at: Date
+  updated_at: Date
+}
+
+export type TopicFeedback = {
+  topic: Topic
```

Comment:
What if this is only `topic_id` instead of topic with an object?

```
type TopicFeedback = {
  topic_id: SurveyTopicAttributes['topic_id']
  reaction: ReactionType
}
```
🤔

Reply:
Isn't this the same?
`export type Topic = Pick<SurveyTopicAttributes, 'topic_id'>`

Reply:
Oh, I can make it 

```typescript
export type TopicFeedback = Topic & {
  reaction: ReactionType
}
```